nipype/utils/draw_gantt_chart.py

```diff
@@ -291,8 +303,8 @@ def draw_nodes(start, nodes_list, cores, minute_scale, space_between_minutes,
             'color': color,
             'node_name': node['name'],
             'node_dur': node['duration'] / 60.0,
-            'node_start': node_start.strftime("%Y-%m-%d %H:%M:%S"),
-            'node_finish': node_finish.strftime("%Y-%m-%d %H:%M:%S")
+            'node_start': node_start,
+            'node_finish': node_finish
```

Comment:
This seems to be replacing a string with a `datetime` object. Is that intended? It will now be formatted with `str(node_start)` and `str(node_finish)` in the format string below:

```Python
new_node = "<div class='node' style='left:%(left)spx;top:%(offset)spx;"\
    ...
    "start:%(node_start)s\nend:%(node_finish)s'></div>" %
    node_dict
```

I'm not sure that's desirable. `strftime` is insensitive to changes in `datetime.__str__` across versions.